frontend/src/core/pyodide/worker/bootstrap.ts

```diff
@@ -41,11 +41,18 @@ export async function bootstrap() {
         "${marimoWheel}",
         "markdown",
         "pymdown-extensions",
+        "pyodide_http",
       ],
       deps=False
       );
     `);
 
+  // Patch the built-in urllib
+  pyodide.runPython(`
```

Comment:
Do you know how long this takes? I assume milliseconds but this could be lazy as well